### DISCUSSION TITLE
Can we prepare training data with PW DFT?

### DISCUSSION THREAD

**Discussion by Youhaojen** [deepmodeling/DeePTB#66] [General]
Hi, can we prepare our own training data with VASP or QE?

**Comment by QG-phy** [comment]
Yes, we can. DeePTB can take eigenvalues from DFT calculations as training labels. One just needs to preprocess the eigenvalues file from the specific DFT package like VASP or QE to the `eigs.npy` file format.

**Comment by Rizwan130** [comment]
Hello, could you please explain how to convert eigenvalues obtained from VASP to eigns.npy? Additionally, how can I convert k-points to kpoints.npy? I've written Python scripts for these conversions, but they're not working as expected. Thank you!

**Comment by QG-phy** [comment]
now dftio + deeptb support to train a sktb model using vasp eigenvalues. pls see the tutorial : https://deepmodeling.github.io/DeePTB/quick_start/hands_on/tutorial2_data.html
or the bohr notebook: 
https://bohrium.dp.tech/notebooks/68191154759
